### ISSUE TITLE
dropout

### ISSUE THREAD

**Issue by InstantWindy** [Lextal/pspnet-pytorch#7]
hello, there is no dropout in paper,if not adding the dropout layer,the network will overfitting?
